packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -201,49 +202,51 @@ export const VtexCommerce = (
             }
           : null
 
+        const addressSession = body?.selectedAddresses?.map((address) => {
```

Comment:
This chunk of code seems to be duplicated, right? Maybe we should move it to a function.

Reply:
Good idea :) Make total sense!